app/views/shared/_header.html.erb

```diff
@@ -0,0 +1,17 @@
+<header class="header">
+  <% if user_signed_in? %>
+    <div class="header-user-container">
+      <div>
+        <%= current_user.name %>
+      </div>
+      <div>
+        <%= current_user.surname %>
+      </div>
+      <%= link_to "Sign out", "/user/sign_out" %>
```

Comment:
I think you can use `destroy_session_path(:user)` or something like that
https://www.rubydoc.info/github/plataformatec/devise/Devise/Controllers/UrlHelpers